src/biopsykit/classification/model_selection/sklearn_pipeline_permuter.py

```diff
@@ -52,6 +53,10 @@ def __init__(
             dictionary has parameters names (str) as keys and lists of parameter settings to try as values, or a list
             of such dictionaries, in which case the grids spanned by each dictionary in the list are explored.
             This enables searching over any sequence of parameter settings.
+        hyper_search_dict : dict, optional
+            Nested dictionary specifying the method for hyperparameter search (e.g., whether to use "grid-search"
```

Comment:
rather use "grid" and "random" instead of "grid-search" and "random-search" to not confuse with the expected dict keys?